features/command_line/example_name_option.feature

```diff
@@ -12,6 +12,8 @@ Feature: `--example` option
   You can also use the option more than once to specify multiple example
   matches.
 
+  Note: description-less examples that have generated descriptions (typical when using the [one-liner syntax](../subject/one-liner-syntax)) cannot be directly filtered with this option, because it is necessary to execute the example to generate the description, so RSpec is unable to use the not-yet-generated description to decide whether or not to execute an example. You can, of course, pass part of a group's description to select all examples defined in the group (including those that have no description).
```

Comment:
It's because relish puts line breaks every where we have line breaks.  I used t use a trial-and-error method of pushing, seeing how it rendered, adjusting line breaks, pushing again, etc.....until I got really annoyed with that workflow and realized that if we don't put any line breaks relish will render it properly.  So while it's annoying in our editors to have this unwrapped it ensures it renders properly on relish without any effort, and it's easy enough to do `:set wrap` (or whatever) in your editor.